frontend/src/employee-frontend/components/invoices/InvoiceFilters.tsx

```diff
@@ -169,4 +174,4 @@ function InvoiceFilters() {
   )
 }
 
-export default InvoiceFilters
+export default React.memo(InvoiceFilters)
```

Comment:
I prefer `export default React.memo(function InvoiceFilters(...) { ... })` but it's a matter of taste I guess 😄